gcloud-java-storage/src/main/java/com/google/gcloud/storage/HttpMethod.java

```diff
@@ -20,5 +20,5 @@
  * Http method supported by Storage service.
  */
 public enum HttpMethod {
-  GET, HEAD, PUT, POST, DELETE
+  GET, HEAD, PUT, POST, DELETE, OPTIONS
```

Comment:
Ok, according to @BrandonY from GCS there is no need to include `TRACE` and `CONNECT`.